js/compare-salary.js

```diff
@@ -0,0 +1,456 @@
+// The VT Check Salary JS
```

Comment:
This whole file is gonna be prime for a refactor. Obviously not now. I'm presuming this works okay, but we might also need to consider wrapping it as an iife or something to do a poor man's bundle of the code.